src/sedtrails/particle_tracer/particle.py

```diff
@@ -16,52 +16,57 @@ class Time:
 
     Attributes
     ----------
-    time_step : int
-        The current time step in a simulation. A non-negative integer.
+    reference_date : numpy.datetime64
+        The reference date for calculating time.
+    offset_seconds : numpy.timedelta64
```

Comment:
Would be more intuitive to pass pass only the number of units to `offset_seconds` and hard code the units (seconds). That way it is easier for humans to interpret. I am thinking that in the configuration file, something like the size of the time step will be defined in seconds, for example. We could also accept both data types, might be a case in which the offset can be process faster if in deltatime

Reply:
Make it an integer.  The class should  be responsible of converting the type is needed